### PR TITLE
Move protobufs to tao/proto.

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

**Comment by kevinawalsh** [comment]
Can you explain the reason for moving these? Is there a benefit?

**Comment by cjpatton** [comment]
This is for internal integration downstream. We'll need access to the protobufs elsewhere and this eliminates a naming conflict in the BUILD script.

File: go/tao/proto/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This directory contains the definitions of protocol buffers used in CloudProxy.
+The `tao` package expects the protobufs to be in the same package; in order to
+build package, they need to be compiled into Go code in the parent directory.
```
Comment (tmroeder) [inline go/tao/proto/README.md]: "build package" -> "build the package"

**Comment by tmroeder** [comment]
With respect to "why move them", I think the second answer is that projects often build proto files into their own library rather than merging them with the main library. I didn't understand that when we started, and I would separate them if I were starting now. I'd view this as a first step in moving in the direction of having a separate library of generated protobuf files, if you don't object, Kevin.

**Comment by kevinawalsh** [comment]
I approve

**Pull request merged**